homeassistant/components/tuya/switch.py

```diff
@@ -433,6 +433,21 @@
             entity_category=ENTITY_CATEGORY_CONFIG,
         ),
     ),
+    # Fingerbot
+    # 
+    "szjqr": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            name="Finger Up",
+            icon="mdi:cursor-pointer",
+        ),
+        SwitchEntityDescription(
+            key=DPCode.TAP_ENABLE,
+            name="Tap Enable",
```

Comment:
Just `Tap`?

```suggestion
            name="Tap",
```

Reply:
I have no idea what this setting does. It doesn't appear in mobile app. Maybe it's better to remove it?

Reply:
In that case, yeah.

Maybe it enables/disables the control of the "finger" from the device?

Reply:
Done